### ISSUE TITLE
Testing process on SauceLabs doesn't complete properly

### ISSUE THREAD

**Issue by Aaronius** [Aaronius/penpal#46]
When tests are running on SauceLabs, the process fails. Here's the last relevant message:

```
09 05 2020 16:58:47.168:WARN [launcher]: Chrome on SauceLabs have not captured in 60000 ms, killing.
```

See an example failed build here: https://travis-ci.org/github/Aaronius/penpal/builds/685098099

I've tried several configuration changes but haven't had success fixing the problem. Alternatively, explore different testing solutions entirely.

**Comment by celesterobinson** [comment]
One option would be to ditch TravisCI and use the [Sauce Labs CLI in conjunction with GitHub actions.](https://docs.saucelabs.com/dev/cli/saucectl/usage/ci/github-actions/).

I'm happy to submit a PR with the workflow, then you can add your Sauce Labs credentials to a GitHub secret.

**Comment by Aaronius** [comment]
Let's do it!

**Comment by celesterobinson** [comment]
Before completely throwing out TravisCI, I'm curious what configuration changes you tried. Did you try launching chrome with no sandbox?

**Comment by Aaronius** [comment]
It's been a few years, so I don't even remember at this point. I believe it was before Github Actions were a thing. I'd rather just use Github Actions at this point.

**Comment by celesterobinson** [comment]
Could you give me permissions to edit actions on the repo while I work on this? I'm not sure which permission it is, but I currently can't re-run jobs :(

**Comment by Aaronius** [comment]
I tweaked a permission. See if that works. If it doesn't, I'll need to do some more research.

**Comment by Aaronius** [comment]
Are you able to re-run jobs now @celesterobinson?

**Comment by celesterobinson** [comment]
Yes! Much thanks. I haven't had time for a while to look at this but I'll have a lot of time next week.

**Comment by celesterobinson** [comment]
@Aaronius I'm not sure if they're already set up, but would you mind setting up the repo secrets?
Keys:
`SAUCE_USERNAME`
`SAUCE_ACCESS_KEY`

**Comment by Aaronius** [comment]
Okay, the secrets are set!

**Comment by Aaronius** [comment]
I'm also good with going with BrowserStack if SauceLabs gives us trouble. I've had better luck with BrowserStack in other projects.

**Comment by Aaronius** [comment]
I'm closing this issue. I've removed SauceLabs usage from version 7 (which is in the pre-release phase). I will likely be adding automation using a different provider later.
